tests/Shipment/ShipmentTest.php

```diff
@@ -97,28 +98,27 @@ public function test_it_should_transform_shipment_to_json_api_with_minimum_requi
         self::assertEquals([
             'type'          => 'shipments',
             'attributes'    => [
-                'recipient_address'   => [
+                'recipient_address'                    => [
```

Comment:
Why the indentation?

Reply:
No idea probably didnt format after changing code. Will fix.

Reply:
I think this happened because I changed something and then didn't format.